packages/components/toggle-group/src/toggle-group.ts

```diff
@@ -0,0 +1,102 @@
+import { RovingTabindexController } from '@sl-design-system/shared';
+import { type SlToggleEvent } from '@sl-design-system/shared/events.js';
+import { ToggleButton, type ToggleButtonSize } from '@sl-design-system/toggle-button';
+import { type CSSResultGroup, LitElement, type PropertyValues, type TemplateResult, html } from 'lit';
+import { property } from 'lit/decorators.js';
+import styles from './toggle-group.scss.js';
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'sl-toggle-group': ToggleGroup;
+  }
+}
+
+/**
+ * A component for visually grouping toggle buttons together.
```

Comment:
maybe also add something about the single/multiple functionality? It's more than just visual